db/migrations/20181114200131_create_vat_storage_tables.sql

```diff
@@ -74,10 +80,11 @@ CREATE INDEX vat_ilk_spot_ilk_index
 CREATE TABLE maker.vat_ilk_rate
 (
     id        SERIAL PRIMARY KEY,
+    diff_id   INTEGER NOT NULL REFERENCES storage_diff (id) ON DELETE CASCADE,
```

Comment:
👍 Nice catch! Updated in 1ad6498.

I'm also wondering if this sort of change is something we'd want to consider for header_id and address_id. Seems like the number of headers and addresses will grow way slower than diffs, but it would probably also be easier to handle the type change now vs. later.

Reply:
Yeah good call, I don't see a reason not to use it for those other fields. Thinking we may need to update https://github.com/makerdao/vulcanizedb/pull/8 so that the actual primary key on the table is a `bigserial`, and would be up for making a similar change to other tables whenever

Reply:
new story: https://makerdao.atlassian.net/browse/VDB-1082